packages/cdc-react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "../Button/Button";
+import "./Card.scss";
+
+interface CardProps {
+  cardLayout: "horizontal" | "vertical-left" | "vertical-right";
```

Comment:
maybe a cardHeaderStyle prop?

Reply:
How do you mean?

Reply:
I had just noticed some of the header styles were different.

Reply:
I believe this was addressed by changing the name of another prop to `setMediaBanner`. The horizontal view can have the title below or above the image based on a prop.